staging/src/k8s.io/apiserver/pkg/endpoints/apiserver_test.go

```diff
@@ -2971,38 +2990,42 @@ func TestUpdate(t *testing.T) {
 }
 
 func TestUpdateInvokesAdmissionControl(t *testing.T) {
-	storage := map[string]rest.Storage{}
-	simpleStorage := SimpleRESTStorage{}
-	ID := "id"
-	storage["simple"] = &simpleStorage
-	handler := handleDeny(storage)
-	server := httptest.NewServer(handler)
-	defer server.Close()
+	for _, admit := range []admission.Interface{alwaysMutatingDeny{}, alwaysValidatingDeny{}} {
```

Comment:
I'm not sure these are checking what we want.  We need to make sure that generic store calls admission.  This is ensuring that our mock calls it.  It's good that our mock calls it, but this is just checking the mock, not the actual code, right?

Reply:
Not on this level. The generic store has its own tests which are more e2e.